doc/release/2.8.1.rst

```diff
@@ -0,0 +1,347 @@
+    (:tarantool-issue:`5536`).
+
+-   The information about the state of synchronous replication is now available via the
+    ``box.info.synchro`` interface (:tarantool-issue:`5191`).
+
+-   Tarantool can now :ref:`run multiple iproto threads <cfg_networking-iproto_threads>`.
+    It is useful in some specific workloads where iproto thread is the bottleneck of
+    throughput (:tarantool-issue:`5645`).
+
+-   Update operations can’t insert with gaps. This patch changes the
+    behavior so that the update operation fills the missing fields with
+    nulls (:tarantool-issue:`3378`).
+
+-   The new ``box.lib`` module allows loading and executing C
+    stored procedures on read-only nodes (:tarantool-issue:`4642`).
+
+-   Configuration options in `box.cfg` can now be defined with environment variables
+    (:tarantool-issue:`5602`).
+
+    The priority of sources of configuration options is the following, from higher to lower:
+
+    *   ``box.cfg{}``,
+    *   environment variables,
+    *   ``tarantoolctl`` options,
+    *   default values,
+
+
+Replication
+~~~~~~~~~~~
+
+-   Introduced the ``box.ctl.promote()`` function and the concept of manual elections
+    (enabled with ``election_mode='manual'``) (:tarantool-issue:`3055`).
+
+    Once the instance is in the ``manual`` election mode,
+    it acts like a ``voter`` most of the time,
+    but may trigger elections and become a leader when ``box.ctl.promote()`` is called.
+    When ``election_mode ~= 'manual'``,
+    ``box.ctl.promote()`` replaces ``box.ctl.clear_synchro_queue()``,
+    which is now deprecated.
+
+
+LuaJIT
+~~~~~~
+
+-   The output of LuaJIT memory profiler parser has become more user-friendly
+    (:tarantool-issue:`5811`). Now the source line definition where the event occurs is
+    much clearer: it only shows the source file name and allocation-related line,
+    and omits the line number of the function definition.
+    Moreover, event-related statistics are now indicated with units.
+
+    **Breaking change**: Line info of the line function definition is now saved in
+    symbol info table by field ``linedefined``.
+    Field ``name`` has been renamed to ``source`` with the respect to the Lua Debug API.
+
+-   A number of improvements in the memory profiler parser:
+
+    *   The parser now reports heap difference which occurs during
+        the measurement interval (:tarantool-issue:`5812`).
+
+    *   Use the option ``--leak-only`` to show only the heap difference.
+
+    *   New built-in module ``memprof.process`` performs post-processing and aggregation of
+        memory events.
+
+    Run the memory profiler with the following command:
+
+    ..  code-block:: bash
+
+        tarantool -e 'require("memprof")(arg)' - --leak-only /tmp/memprof.bin``
+
+
+Tools
+~~~~~
+
+-   New tooling for collecting crash artefacts and postmortem analysis
+    (:tarantool-issue:`5569`).
+
+
+Build
+~~~~~
+
+-   Tarantool build infrastructure now requires CMake version 3.1 or later.
```

Comment:
It is 3.2 due to https://github.com/tarantool/tarantool/pull/5988 (it was pushed to the same set of releases). My bad, I didn't updated the changelog entry.